packages/netstring/src/reader.js

```diff
@@ -0,0 +1,71 @@
+// @ts-check
```

Comment:
this overlaps substantially with https://github.com/Agoric/agoric-sdk/blob/master/packages/xsnap/src/netstring.js 
 62e0d5a  Feb 10 ... manually checking to see how it's different is tedious. Checking in netstring.js 62e0d5a and then changing it would have let me use tools.